Sources/HummingbirdJobs/JobQueue.swift

```diff
@@ -55,7 +55,7 @@ public struct JobQueue<Queue: JobQueueDriver>: Service {
     ///   - maxRetryCount: Maximum number of times job is retried before being flagged as failed
     ///   - execute: Job code
     public func registerJob<Parameters: Codable & Sendable>(
-        _ id: JobIdentifier<Parameters>,
+        id: HBJobIdentifier<Parameters>,
```

Comment:
HB prefix is in there?
```suggestion
        id: JobIdentifier<Parameters>,
```

Reply:
shit I shouldn't rush these things